src/components/card/card.module.css

```diff
@@ -0,0 +1,43 @@
+.cardDefault{
```

Comment:
```suggestion
.card {
```

Lembre do espaçamento.